pkgs/shells/bash/bash-completion/default.nix

```
@@ -11,11 +11,13 @@ stdenv.mkDerivation rec {
   pname = "bash-completion";
   version = "2.11";
 
-  src = fetchFromGitHub {
-    owner = "scop";
-    repo = "bash-completion";
-    rev = version;
-    sha256 = "0m3brd5jx7w07h8vxvvcmbyrlnadrx6hra3cvx6grzv6rin89liv";
+  # Using fetchurl because fetchGithub or fetchzip will have trouble on
+  # e.g. APFS filesystems (macOS) because of non UTF-8 characters in some of the
+  # test fixtures that are part of the repository.
+  # See discussion in https://github.com/NixOS/nixpkgs/issues/107768
+  src = fetchurl {
+    url = "https://github.com/scop/${pname}/releases/download/${version}/bash-completion-${version}.tar.xz";
```

Minor nit-pick, otherwise ACK

```suggestion
    url = "https://github.com/scop/${pname}/releases/download/${version}/${pname}-${version}.tar.xz";
```

---

Good catch